src/Razor/src/Microsoft.VisualStudio.LiveShare.Razor/Guest/GuestProjectPathProvider.cs

```diff
@@ -92,7 +91,7 @@ internal virtual Uri GetHostProjectPath(ITextDocument textDocument)
                 var projectHierarchyProxy = _proxyAccessor.GetProjectHierarchyProxy();
 
                 // We need to block the UI thread to get a proper project path. However, this is only done once on opening the document.
-                return projectHierarchyProxy.GetProjectPathAsync(ownerPath, CancellationToken.None);
+                return projectHierarchyProxy.GetProjectPathAsync(ownerPath!, CancellationToken.None);
```

Comment:
Again, an assert or assumes will help diagnose issues where this assumption is wrong.

I feel like every null suppression, unless it's dotted off, should probably have one, and/or a comment.